### PR TITLE
Bug # 22683 : PlayFabSetting.LogLevel in the Unity SDK does nothing

### PR DESCRIPTION
**Pull request by JTonoJimenez**

### CODE REVIEW

File: targets/unity-v2/source/Shared/Internal/Log.cs [hunk 1]
```diff
@@ -1,7 +1,10 @@
+using System;
+
 namespace PlayFab.Internal
 {
     public static class Log
     {
+        [Obsolete("Debug has deprecated.")]
```
Comment (toddbellMSFT) [inline targets/unity-v2/source/Shared/Internal/Log.cs]: I think either "Debug has been deprecated." or "Debug is deprecated." would flow a bit easier.

File: targets/unity-v2/source/Shared/Internal/Log.cs [hunk 2]
```diff
@@ -10,6 +13,7 @@ public static void Debug(string text, params object[] args)
             }
         }
 
+        [Obsolete("Info has deprecated.")]
```
Comment (toddbellMSFT) [inline targets/unity-v2/source/Shared/Internal/Log.cs]: nitpick: "Info is deprecated." or "Info has been deprecated."

File: targets/unity-v2/source/Shared/Internal/Log.cs [hunk 3]
```diff
@@ -18,6 +22,7 @@ public static void Info(string text, params object[] args)
             }
         }
 
+        [Obsolete("Warning has deprecated.")]
```
Comment (toddbellMSFT) [inline targets/unity-v2/source/Shared/Internal/Log.cs]: nitpick, similar idea.

**Comment by JTonoJimenez** [comment]
retest

**Comment by pgilmorepf** [comment]
@JTonoJimenez please add all four of the reviewers to all of your code reviews:
@pgilmorepf @Vicodex @MichaelMackendMS @toddbellMSFT

**Comment by JTonoJimenez** [comment]
retest

File: targets/unity-v2/source/Shared/Public/PlayFabSettings.cs.ejs [hunk 4]
```diff
@@ -97,6 +97,7 @@ namespace PlayFab
         /// These are variables which are always singleton global
         /// </summary>
         #region PlayFabSharedSettings Redirects
+		[ObsoleteAttribute("LogLevel has been deprecated, please use UnityEngine.Debug.Log for your logging needs.")]
```
Comment (pgilmorepf) [inline targets/unity-v2/source/Shared/Public/PlayFabSettings.cs.ejs]: tabs

**Comment by JTonoJimenez** [comment]
retest

File: targets/unity-v2/source/Shared/Internal/Log.cs [hunk 5]
```diff
@@ -1,7 +1,10 @@
+using System;
+
 namespace PlayFab.Internal
 {
     public static class Log
```
Comment (pgilmorepf) [inline targets/unity-v2/source/Shared/Internal/Log.cs]: Let's deprecate the whole class:
```suggestion
    [Obsolete("This logging utility has been deprecated. Use UnityEngine.Debug.Log")]
    public static class Log
```

**Comment by pgilmorepf** [comment]
Are there other lingering logging classes? Part of the problem was that there was more than 1.

**Comment by JTonoJimenez** [comment]
retest

**Comment by pgilmorepf** [comment]
retest

**Comment by pgilmorepf** [comment]
This branch was forked from an old version of master, which wasn't yet fixed for switch. I've rebased it, and it should pass this time.

**Pull request merged**